src/tests/__mocks__/dictionaries.js

```diff
@@ -46,4 +46,22 @@ export const mockDictionaries = (() => {
   return dictionaries;
 })();
 
+export const threeMockDictionaries = [
+  dictionary,
```

Comment:
Are these three or two?

Reply:
Those are *three*. `dictionary` on line 50 is also a dictionary. The other two are created from that first one with a few changes to avoid code repetition.